<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinecraftPage</class>
 <widget class="QWidget" name="MinecraftPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>1134</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="minecraftTab">
      <attribute name="title">
       <string notr="true">Minecraft</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="windowSizeGroupBox">
         <property name="title">
          <string>Window Size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="maximizedCheckBox">
            <property name="text">
             <string>Start Minecraft maximized?</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayoutWindowSize">
            <item row="1" column="0">
             <widget class="QLabel" name="labelWindowHeight">
              <property name="text">
               <string>Window hei&amp;ght:</string>
              </property>
              <property name="buddy">
               <cstring>windowHeightSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelWindowWidth">
              <property name="text">
               <string>W&amp;indow width:</string>
              </property>
              <property name="buddy">
               <cstring>windowWidthSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="windowWidthSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65536</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>854</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="windowHeightSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65536</number>
              </property>
              <property name="value">
               <number>480</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="nativeLibWorkaroundGroupBox">
         <property name="title">
          <string>Native library workarounds</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="useNativeGLFWCheck">
            <property name="text">
             <string>Use system installation of GLFW</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useNativeOpenALCheck">
            <property name="text">
             <string>Use system installation of OpenAL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerMinecraft">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>maximizedCheckBox</tabstop>
  <tabstop>windowWidthSpinBox</tabstop>
  <tabstop>windowHeightSpinBox</tabstop>
  <tabstop>useNativeGLFWCheck</tabstop>
  <tabstop>useNativeOpenALCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
